docs/dev/apis/lethallib/custom-enemies/blender.md

```diff
@@ -0,0 +1,88 @@
+    - Includes a lot of useful information about rigging, but it's a very fast paced video. Can be fairly hard to follow for a complete beginner. Likely a better watch after you've seen the more basic introductory type tutorials first.
+- [How to Rig and Animate in BLENDER!](https://youtu.be/1khSuB6sER0) - ProductionCrate
+    - Learn how to make a rig for a humanoid character, fix issues with Blender's automatic weights feature, as well as inverse kinematics. 
+
+## Animation & NLA (Nonlinear Animation) Editor
+
+::: info 
+We should put our individual animations in the NLA Editor so we can use them separately in Unity. The length of the animation in Unity will be the length that you set in the NLA editor. This is important to know if you set an animation cycle to repeat a certain amount of times in Blender when you want to for example preview it in combination with your other animations.
+:::
+
+- [The Nuts and Bolts of Blender's animation system](https://youtu.be/p3m57yAcsi0) - CGDive
+    - Introduces concepts in a very in-depth way. Introduces Timeline, Dope Sheet, Graph Editor, NLA Editor, Actions.
+- [Un-confusing the NLA Editor (Nonlinear Animation)](https://youtu.be/tAo7HxxxA08) - GCDive
+    - A more in-depth video about the NLA Editor. Do note though, we do not need to do anything complex with the NLA Editor.
+- [Become a PRO at Animation in 25 Minutes | Blender Tutorial](https://youtu.be/_C2ClFO3FAY) - CG Geek
+    - Animating a walk cycle. Uses Timeline, Dope Sheet and Graph Editor. Uses references for animation.
+- [Character animation for impatient people - Blender Tutorial](https://youtu.be/GAIZkIfXXjQ) - Joey Carlino
+    - If you don't want to make and rig your own models.
+
+### Animation - Common Issues
+
+**Objects in Model appear in different places as in Blender when exporting to Unity.**
+This might be because you have directly animated an object, instead of an armature. Try parenting your object to an armature and remake your animation with that.
+
+**Animations are broken in Unity.**
+This might be because you have animations with the same name in your NLA editor. Make sure your animations have unique names.
+
+## Exporting Assets For Unity
+
+To export your model, go to: `File` -> `Export` -> `FBX (.fbx)`  
+This will open our FBX exporter window, where we have some options available to us. The most important thing here however is the transform section. Because of the differences in Blender's and Unity's coordinate systems, exporting your model is not quite as straight-forward as you'd think. It's very easy to get your model pointing in the wrong direction, being sideways, or even upside down if you don't have correct values set.
+
+Even I don't understand how any it really works, but our model points in the negative Z direction, and in the transfrom section, we have set `Forward` to `-Z Forward`, and `Up` to `Y Up` and our model appears correctly in Unity.  
```

Comment:
Unity uses a left-handed, Y-Up coordinate system, whereas Blender uses a right-handed, Z-Up coordinate system. The transformation you do changes Blender's frame of reference to match the coordinate system Unity uses.